packages/xo-server/src/fatfs-buffer.mjs

```diff
@@ -23,6 +23,28 @@ const SECTOR_SIZE = 512
 
 const TEN_MIB = 10 * 1024 * 1024
 
+// https://en.wikipedia.org/wiki/Master_boot_record
+// we'll add a classical generic mbr
+// with one FAT16 partition addressed by LBA
+
+export function addMbr(buf) {
+  const mbr = Buffer.alloc(SECTOR_SIZE, 0)
+  // 0 - 446 is bootstrap code , keep it empty
+
+  // entry
+  mbr[446] = 0x08 // entry is bootable
```

Comment:
I don't understand this value, from what I read (the 7th bit is set to `1`), shouldn't it be `0x80`?